pkg/Library/NodeGraph/dom/node-graph.js

```diff
@@ -247,8 +247,15 @@ export class NodeGraph extends Xen.Async {
   }
 
   // called when committed a change to a rectangle (low freq)
-  onUpdatePosition({currentTarget: {value: rect}}) {
-    this.fire('node-moved');
+  // onUpdatePosition({currentTarget: {value: rect}}) {
+  //   this.fire('node-moved');
+  // }
+  onUpdatePosition({currentTarget: {target, value}}) {
+    if (target?.key && value) {
```

Comment:
could you, please, take a look and see if this change looks reasonable? i'm not fully confident in my understanding of how node-graph events work...